pdpdmeetup/templates/ajax/ajax_meetups.html

```diff
@@ -16,8 +16,11 @@
                         <tr>
                             <td class="event-table-col1">When?</td>
                             <td>
-                                <b title="{{ each_event.event_datetime|date:"T" }}">{{ each_event.event_datetime|date:"l, d M Y, P" }}
```

Comment:
I'm not sure why this info was in a <b title=""> so i pulled it out... seems to work much the same.